src/styles/blocks/cards.scss

```
@@ -0,0 +1,98 @@
+.card {
+  width: $card-width;
+  margin-right: $card-margin-right;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__image {
+    margin-bottom: 40px;
+    display: flex;
+    justify-content: center;
+  }
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    width: $card-width-inside;
+    margin-bottom: 4px;
+    font-size: $font-size-normal;
+    font-weight: $font-weight-normal;
+    line-height: 1.5;
+    color: $color-text;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+    font-size: $font-size-small;
+    line-height: 14px;
+    font-weight: $font-weight-small;
+    color: $color-text-gray;
+  }
+
+  &__rating {
+    width: $card-width-inside;
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price {
+    width: $card-width-inside;
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__text-price {
+    font-size: $font-size-normal;
+    font-weight: $font-weight-small;
+    color: $color-text-gray;
+  }
+
+  &__money {
+    font-size: $font-size-large;
+    font-weight: $font-weight-big;
+    line-height: 18px;
+    color: $color-text;
+  }
+
+  &__button-text {
+    width: $card-width-inside;
+    padding: 10.8px 69px;
+    display: flex;
+    font-family: inherit;
+    font-size: $font-size-big;
+    font-weight: $font-weight-big;
+    color: #fff;
+    background-color: $color-button;
+    border: 1px solid $color-button;
+    border-radius: 5px;
+    justify-content: center;
+  }
+  
+  &__button-text:hover {
+    color: $color-button;
+    background-color: #fff;
+  }
+

```

Use nesting for BEM elements in scss. Fix all cases please
```suggestion
  &__button-text {
    width: $card-width-inside;
    padding: 10.8px 69px;
    display: flex;
    font-family: inherit;
    font-size: $font-size-big;
    font-weight: $font-weight-big;
    color: #fff;
    background-color: $color-button;
    border: 1px solid $color-button;
    border-radius: 5px;
    justify-content: center;
    
    &:hover {
      color: $color-button;
      background-color: #fff;
    }
  }
```